kani-driver/src/call_cbmc.rs

```diff
@@ -404,6 +425,60 @@ fn determine_failed_properties(properties: &[Property]) -> FailedProperties {
     }
 }
 
+fn coverage_results_from_properties(properties: &[Property]) -> Option<CoverageResults> {
+    let cov_properties: Vec<&Property> =
+        properties.iter().filter(|p| p.is_code_coverage_property()).collect();
+
+    if cov_properties.is_empty() {
+        return None;
+    }
+
+    let counter_re = {
+        static COUNTER_RE: OnceLock<Regex> = OnceLock::new();
+        COUNTER_RE.get_or_init(|| {
+            Regex::new(
+                r#"^(?<kind>CounterIncrement|ExpressionUsed)\((?<counter_num>[0-9]+)\) \$(?<func_name>[^\$]+)\$ - (?<span>.+)"#,
```

Comment:
Can we add an example pattern this regex is matching against?

Reply:
Done. I've also linked to #3445 in the comment.